conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureUndertowRuntime.java

```diff
@@ -76,6 +77,11 @@ public AsyncRequestProcessing async() {
         return async;
     }
 
+    @Override
+    public ExceptionHandler exceptions() {
```

Comment:
I think we'll want to expose this on the builder

Reply:
:+1: